html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <form action="">
+      <h1>Coding Form!</h1>
+      <div>Email</div>
```

Comment:
Make sure to use `<label>` elements for the text related to your inputs. The label should have a `for` attribute that matches the `id` of the related input.